app/services/state_file/state_information_service.rb

```diff
@@ -83,49 +83,55 @@ def state_intake_class_names
         w2_include_local_income_boxes: false
       },
       id: {
+        mail_voucher_address: "Idaho State Tax Commission<br/>" \
```

Comment:
I see that is getting called on the return status page, we never show the addresses in Spanish right? I don't think we do but might want to check in with product on that if they haven't already said something